sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -113,9 +113,9 @@ export interface EventHubConsumerOptions {
 
 // @public
 export class EventHubProducerClient {
-    constructor(host: string, eventHubName: string, credential: TokenCredential, options?: EventHubClientOptions);
-    constructor(connectionString: string, eventHubName: string, options?: EventHubClientOptions);
     constructor(connectionString: string, options?: EventHubClientOptions);
+    constructor(connectionString: string, eventHubName: string, options?: EventHubClientOptions);
```

Comment:
just to confirm, what version of node were these generated with? v10?

Reply:
This fix ensures the ordering is the same on both v10 and v12.  I generated these files on v12, then re-ran the build with v10 to ensure they didn't change.

Reply:
Awesome!